mobile/src/components/Button.tsx

```diff
@@ -27,8 +27,8 @@ export default function Button({
   ...props
 }: Props) {
   //warning: do not touch the extra spaces at the beginning or it will break the styling
-  let baseStyle = "h-7 items-center justify-center";
-  let textStyle = "text-white margin mx-5 font-semibold";
+  let baseStyle = "h-10 items-center justify-center";
```

Comment:
I didn't catch this before but instead of changing the height I believe what we want is to add padding so that the button content is far enough from the edges. (something like 'px-4 py-2')